src/dbt/kipptaf/models/kippadb/base/base_kippadb__application.sql

```diff
@@ -65,6 +65,7 @@ with
 
             acc.name as account_name,
             acc.type as account_type,
+            acc.adjusted_6_year_minority_graduation_rate,
```

Comment:
@anthonygwalters did you intend to use this in the dashboard view or just adding for other purposes?

Reply:
Needed for ad-hoc stuff and dashboard eventually